# SPDX-License-Identifier: Apache-2.0
diff --git a/src/solvers/CMakeLists.txt b/src/solvers/CMakeLists.txt
index 003a0d957b..79751ef8b2 100644
--- a/src/solvers/CMakeLists.txt
+++ b/src/solvers/CMakeLists.txt
@@ -120,16 +120,6 @@ foreach(SOLVER ${sat_impl})
     elseif("${SOLVER}" STREQUAL "cadical")
         message(STATUS "Building solvers with cadical")

-        download_project(PROJ cadical
-            URL https://github.com/arminbiere/cadical/archive/rel-2.0.0.tar.gz
-            PATCH_COMMAND patch -p1 -i ${CBMC_SOURCE_DIR}/scripts/cadical-2.0.0-patch
-            COMMAND cmake -E copy ${CBMC_SOURCE_DIR}/scripts/cadical_CMakeLists.txt CMakeLists.txt
-            COMMAND ./configure
-            URL_MD5 9fc2a66196b86adceb822a583318cc35
-        )
-
-        add_subdirectory(${cadical_SOURCE_DIR} ${cadical_BINARY_DIR})
-
         target_compile_definitions(solvers PUBLIC
             SATCHECK_CADICAL HAVE_CADICAL
         )
@@ -137,6 +127,7 @@ foreach(SOLVER ${sat_impl})
         target_include_directories(solvers
             PUBLIC
             ${cadical_SOURCE_DIR}/src
+            ${cadical_INCLUDE_DIR}
         )

         target_link_libraries(solvers cadical)
